packages/trax/trax.0.4.0/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "martin@mjambon.com"
+authors: ["Martin Jambon"]
+homepage: "https://github.com/mjambon/trax"
+bug-reports: "https://github.com/mjambon/trax/issues"
+dev-repo: "git+https://github.com/mjambon/trax.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "2.8"}
```
See https://github.com/ocaml/dune/issues/2147 for the build flag and your dune-project file for the version requirement